include/TrustWalletCore/TWBase.h

```diff
@@ -14,6 +14,9 @@
 #endif
 #endif
 
+// Marker to make methods visible to dart ffi
```

Comment:
```suggestion
// Marker for default  visibility
```

Reply:
I don't know which functions use methods in `TWData.h` but I was able to generate wallets / addresses, sign transactions, verify addresses/mnemonic for different blockchains without any lookup errors. 

Do you know a function I can test on flutter android that uses methods in `TWData.h` to make sure?

Reply:
just add same marker to `TWData.h`, `TWString` / `TWData` is just `const void`, others interfaces are good